packages/ra-ui-materialui/src/field/ReferenceManyField.spec.tsx

```diff
@@ -268,4 +273,145 @@ describe('<ReferenceManyField />', () => {
             await screen.findByLabelText('ra.navigation.next');
         });
     });
+
+    describe('"Select all" button', () => {
+        it('should be displayed if an item is selected', async () => {
```

Comment:
this description doesn't correspond to the button behavior. It should be displayed if all the items of the page are selected.